mmv1/templates/terraform/encoders/vertex_ai_featurestore_entitytype_feature.go.erb

```diff
@@ -0,0 +1,8 @@
+if v, ok := d.GetOk("entitytype"); ok {
+	re := regexp.MustCompile("projects/(.+)/locations/(.+)/featurestores/(.+)$")
```

Comment:
Same question as above, do we need `entityTypes`?

Reply:
I've updated it in 047a83b.